fstat/static/js/snackbar.js

```diff
@@ -0,0 +1,17 @@
+$("body").prepend('<div id="snackbar"></div>')
```

Comment:
All templates are derived from base.html, isn't it? So you'll only add it on one template.